source/locales.tex

```diff
@@ -2573,64 +2266,62 @@
   ct == use_facet<numpunct<charT>>(loc).thousands_sep()
   && use_facet<numpunct<charT>>(loc).grouping().length() != 0;
 \end{codeblock}
-where the values
-\tcode{src}
-and
-\tcode{atoms}
-are defined as if by:
+where the values \tcode{src} and \tcode{atoms} are defined as if by:
 \begin{codeblock}
 static const char src[] = "0123456789abcdefxABCDEFX+-";
 char_type atoms[sizeof(src)];
 use_facet<ctype<charT>>(loc).widen(src, src + sizeof(src), atoms);
 \end{codeblock}
-for this value of
-\tcode{loc}.
+for this value of \tcode{loc}.
 
-If \tcode{discard} is \tcode{true}, then if
-\tcode{'.'}
-has not yet been accumulated, then the position of the character is remembered,
+If \tcode{discard} is \tcode{true},
+then if \tcode{'.'} has not yet been accumulated,
+then the position of the character is remembered,
 but the character is otherwise ignored.
-Otherwise, if
-\tcode{'.'}
-has already been accumulated, the character is discarded and
-Stage 2 terminates.
-If it is not discarded, then a check is made to determine if \tcode{c} is
-allowed as the next character of an input field of the conversion specifier
-returned by Stage 1. If so, it is accumulated.
-
-If the character is either discarded or accumulated then \tcode{in}
-is advanced by
-\tcode{++in}
+Otherwise, if \tcode{'.'} has already been accumulated,
+the character is discarded and Stage 2 terminates.
+If it is not discarded,
+then a check is made to determine
+if \tcode{c} is allowed as the next character of
+an input field of the conversion specifier returned by Stage 1.
+If so, it is accumulated.
+
+If the character is either discarded or accumulated
+then \tcode{in} is advanced by \tcode{++in}
 and processing returns to the beginning of stage 2.
 
 \stage{3}
-The sequence of \tcode{char}{s} accumulated in stage 2 (the field) is converted to a numeric value by the rules of one of the functions declared in the header \libheader{cstdlib}:
+The sequence of \tcode{char}{s} accumulated in stage 2 (the field)
+is converted to a numeric value by the rules of one of the functions
+declared in the header \libheader{cstdlib}:
 
 \begin{itemize}
-\item For a signed integer value, the function \tcode{strtoll}.
-
-\item For an unsigned integer value, the function \tcode{strtoull}.
-
-\item For a \tcode{float} value, the function \tcode{strtof}.
-
-\item For a \tcode{double} value, the function \tcode{strtod}.
-
-\item For a \tcode{long double} value, the function \tcode{strtold}.
+\item
+For a signed integer value, the function \tcode{strtoll}.
+\item
+For an unsigned integer value, the function \tcode{strtoull}.
+\item
+For a \tcode{float} value, the function \tcode{strtof}.
+\item
+For a \tcode{double} value, the function \tcode{strtod}.
+\item
+For a \tcode{long double} value, the function \tcode{strtold}.
```

Comment:
Why not indent each `\item` text by 2 spaces [as required](https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines), or just remove the original blank lines between items?

Reply:
I don't think we've made up our mind on `\item` source formatting just yet.

Reply:
@zygoloid was ultimately against formatting that was difficult to relocate or regroup, so I think we may be moving away from that rule.